rubrik_cdm/cluster.py

```diff
@@ -602,3 +602,19 @@ def read_only_authorization(self, username, timeout=15):
 
         self.log("read_only_authorization: Granting read-only privilages to user '{}'.".format(username))
         return self.post("internal", "/authorization/role/read_only_admin", config, timeout)
+
+    def get_all_vcenters(self, timeout=60):
+        """Thie function will call the cluster API and get a list of all hosts within the cluster.
+        
+        Arguments:
+            None
+        Returns:
+            dict -- the full API response for `GET /vmware/vcenter`
+        
+        """
+
+        self.log('get_all_vcenters: getting a list of all hosts in the Rubrik cluster.')
+        try:
+            current_hosts = self.get('v1', '/vmware/vcenter', timeout=timeout)  
+        except SystemExit as error:
```

Comment:
You don't need to include the try/except here. That will automatically be handled in the `get()`.

You can also just do a `return self.get('v1', '/VMware/vcenter', timeout=timeout)`

Reply:
Also, what's the end piece of information we should be providing here? I think it would be worthwhile to filter that out instead of providing the full API response. For example, return a list with all vCenter Names and ID or something similar.